### PR TITLE
modify package.json to reference this new forked repo

### PR DESCRIPTION
**Pull request by nickshine**
Setting up initial `package.json` for the new fork.

### CODE REVIEW

**Comment by nickshine** [comment]
@LongLiveCHIEF how do you want to be listed in contributors?

**Comment by nickshine** [comment]
@LongLiveCHIEF You want to work off a development branch or PRs straight to master?

**Comment by nickshine** [comment]
closes #1

File: package.json [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+      "name": "Hakim El Hattab",
+      "email": "hakim.elhattab@gmail.com",
+      "web": "http://hakim.se"
+    },
+    {
+      "name": "Nick Shine",
+      "email": "shine.nick@gmail.com",
+      "url": "https://nshine.dev"
+    }
+  ],
```
Comment (LongLiveCHIEF) [inline package.json:28]:
```
{
  "name": "Brian Vanderbusch",
  "email": "chief@hackerhappyhour.com",
  "web": "http://github.com/LongLiveCHIEF"
}
```

File: package.json [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
   "repository": {
     "type": "git",
-    "url": "git://github.com/hakimel/reveal.js.git"
+    "url": "https://github.com/HackerHappyHour/reveal.js.git"
   },
+  "bugs": "https://github.com/HackerHappyHour/reveal.js/issues",
   "engines": {
```
Comment (LongLiveCHIEF) [inline package.json:34]: Let's update this to LTS versions of node.

**Comment by LongLiveCHIEF** [comment]
@nickshine good question. In most cases I would say direct to master, but there's some reason to argue for a dev branch on this one.

I think my vote will still be direct to `master` though.

File: package.json [hunk 3]
```diff
@@ -18,16 +18,21 @@
     {
       "name": "Nick Shine",
       "email": "shine.nick@gmail.com",
-      "url": "https://nshine.dev"
-    }
+      "url": "https://https://github.com/nickshine"
+    },
+		{
```
Comment (LongLiveCHIEF) [inline package.json]: formatting bugged here.

Comment (cujarrett) [reply package.json]: 👍

**Review by cujarrett — COMMENTED** [review]
Looks good, just one comment on the formatting for the Brian contributor being indented too far.

**Pull request merged**